### PR TITLE
use versioned obsoletes for proxy plugins

### PR DESCRIPTION
**Pull request by evgeni**
otherwise this confuses RPM when upgrading from EL7 (SCL) to EL8 (non-SCL) and both packages are available for a brief moment of time

### CODE REVIEW

File: packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec [hunk 1]
```diff
@@ -49,7 +49,7 @@ Group: Documentation
 Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
-%{?scl:Obsoletes: rubygem-%{gem_name}-doc}
+%{?scl:Obsoletes: rubygem-%{gem_name}-doc < %{version}-%{release}}
```
Comment (ekohl) [inline packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec:52]: It's surprising to me that this has an obsoletes for the -doc package (if SCL) but not for the main package. It feels to me that we should have both or neither. To be consistent, I'd say we should have both.

**Review by ekohl — APPROVED** [review]

File: packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec (hunk 1 above)
Comment (evgeni) [reply packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec:52]: I think it was added *after* the SCL-ization of the proxy, and for some reason we removed the non-doc, but kept the doc one…

Comment (evgeni) [reply packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec:52]: proof: c8588a0e8487cb2010be21974fcbc98c418294d7

Comment (ekohl) [reply packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec:52]: That makes sense.

Comment (evgeni) [reply packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec:52]: so, cleaning this up, I'd just drop the obsoletes from ACD completely? as it really not required there?

**Comment by evgeni** [comment]
Repoclosure is because of Salt, which is expected (it's currently untagged exactly because of that, and I'll untag again)

File: packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec (hunk 1 above)
Comment (ekohl) [reply packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec:52]: Yes, just drop it indeed.

**Review by ekohl — APPROVED** [review]

**Pull request merged**